### PR TITLE
initial codecov config with codecov yml

### PR DESCRIPTION
**Pull request by jeffvg**
#### Changes proposed in this Pull Request:
- add codecov and configs with codecod yml
- 
- 
https://app.codecov.io/gh/pxblue/react-native-workflows

### CODE REVIEW

File: login-workflow/codecov.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+codecov:
+  require_ci_to_pass: yes
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
```
Comment (joebochill) [inline login-workflow/codecov.yml:6]: What does this 70-100 configuration do? Is this going to skip things that have less than 70% coverage?

File: .circleci/config.yml [hunk 2]
```diff
@@ -77,3 +91,10 @@ workflows:
               only:
                 - master
                 - dev
+      - coverage_report:
+                   requires:
+                        - build_login_workflow
+                  #  filters:
+                  #       branches:
+                  #           only:
+                  #               - master
```
Comment (joebochill) [inline .circleci/config.yml:100]: Did you intend to leave this filter commented out?

Comment (jeffvg) [reply .circleci/config.yml]: I just updated, should be good now

**Review by joebochill — APPROVED** [review]

File: .circleci/config.yml [hunk 3]
```diff
@@ -46,6 +48,18 @@ jobs:
...
+        working_directory: ~/react-native-workflows
+        docker:
+            - image: circleci/node:12.9.1-browsers
+        steps:
+            - checkout
+            - attach_workspace:
+                at: .
+            - codecov/upload:
+                file: './login-workflow/coverage/clover.xml'
+                token: 6bd0adf7-ec9c-43ad-b1e9-dd71013923f2
```
Comment (huayunh) [inline .circleci/config.yml:62]: We cool leaving the token here?

**Pull request merged**